patch-package
--- a/node_modules/detox/src/ios/expect.js
+++ b/node_modules/detox/src/ios/expect.js
@@ -290,16 +290,19 @@ class Element {
   async scroll(amount, direction = 'down') {
     // override the user's element selection with an extended matcher that looks for UIScrollView children
     this._selectElementWithMatcher(this._originalMatcher._extendToDescendantScrollViews());
+    this.atIndex(0);
     return await new ActionInteraction(this, new ScrollAmountAction(direction, amount)).execute();
   }
   async scrollTo(edge) {
     // override the user's element selection with an extended matcher that looks for UIScrollView children
     this._selectElementWithMatcher(this._originalMatcher._extendToDescendantScrollViews());
+    this.atIndex(0);
     return await new ActionInteraction(this, new ScrollEdgeAction(edge)).execute();
   }
   async swipe(direction, speed = 'fast', percentage = 0) {
     // override the user's element selection with an extended matcher that avoids RN issues with RCTScrollView
     this._selectElementWithMatcher(this._originalMatcher._avoidProblematicReactNativeElements());
+    this.atIndex(0);
     return await new ActionInteraction(this, new SwipeAction(direction, speed, percentage)).execute();
   }
   async setColumnToValue(column,value) {
