patch-package
--- a/node_modules/react-native-navigation/lib/ios/RNNBottomTabsOptions.m
+++ b/node_modules/react-native-navigation/lib/ios/RNNBottomTabsOptions.m
@@ -49,6 +49,10 @@ - (void)applyOn:(UIViewController *)viewController {
 		viewController.tabBarController.tabBar.clipsToBounds = [self.hideShadow boolValue];
 	}

+	 if (self.visible) {
+    [viewController.tabBarController.tabBar setHidden:![self.visible boolValue]];
+  }
+
 	[self resetOptions];
 }

