patch-package
--- a/node_modules/react-native/react.gradle
+++ b/node_modules/react-native/react.gradle
@@ -103,7 +103,7 @@ afterEvaluate {
                     config."bundleIn${buildTypeName.capitalize()}" ?:
                             targetName.toLowerCase().contains("release")

-                if (isAndroidLibrary) {
+                // if (isAndroidLibrary) {
                     doLast {
                         def moveFunc = { resSuffix ->
                             File originalDir = file("${resourcesDir}/drawable-${resSuffix}")
@@ -119,7 +119,7 @@ afterEvaluate {
                         moveFunc.curry("xxhdpi").call()
                         moveFunc.curry("xxxhdpi").call()
                     }
-                }
+                // }
             }

             // Hook bundle${productFlavor}${buildType}JsAndAssets into the android build process
